modules/core/shared/src/main/scala/gem/enum/GsaoiOdgwSize.scala

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import cats.instances.string._
+import cats.syntax.eq._
+import gem.util.Enumerated
+
+/**
+ * Enumerated type for Gsaoi ODGW Size.
+ * @group Enumerations (Generated)
+ */
+sealed abstract class GsaoiOdgwSize(
+  val tag: String,
+  val size: Int
```

Comment:
I suppose that is size as in pixels?

Reply:
The old code base doesn't say anything about it nor the UI as far as I can tell

Reply:
This seems like thee kind of thing we should be documenting now, so that we avoid this issue resurfacing.

Reply:
This is generated code so we can't document it, but I've renamed it to `pixels` and renamed the table too